contracts/src/v0.8/functions/dev/1_0_0/interfaces/IFunctionsSubscriptions.sol

```diff
@@ -25,6 +25,15 @@ interface IFunctionsSubscriptions {
   // @return subscription - see IFunctionsSubscriptions.Subscription for more information on the structure
   function getSubscription(uint64 subscriptionId) external view returns (Subscription memory);
```

Comment:
Did you consider removing getSubscription and maybe reverting some of the other, less-obvious, gas savings?

Reply:
Which ones do you find most confusing?

I'm thinking we try to get the Invariant revert back in

Reply:
To me that loop change where I commented "what happened here?" is pretty confusing.

Reply:
I agree. The loop is fine.  I am more concerned with the balance invariant